src/app/app.module.ts

```diff
@@ -2,10 +2,14 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './general-components/header/header.component';
```

Comment:
Is importing header & footer components 100% necessary in app.module?

Reply:
Just updated this. Could you check?

Reply:
Seems much better!